app/src/main/java/com/github/studydistractor/sdp/event/EventModel.kt

```diff
@@ -8,40 +8,6 @@ import com.google.android.gms.tasks.Task
  * Handle storage and retrieval of events for EventScreen
  */
 interface EventModel {
-    /**
```

Comment:
Why do you need to remove all these functions ?

Reply:
Because they weren't used nor useful at all as the code works fine without it